resources/assets/js/videos/components/videoScreen/drawInteractions.vue

```diff
@@ -169,10 +169,11 @@ export default {
                 this.$emit('is-invalid-shape', e.feature.getGeometry().getType());
                 // Wait for this feature to be added to the source, then clear.
                 this.pendingAnnotationSource.once('addfeature', () => {
-                    this.resetPendingAnnotation();
+                    // Clear annotation source only, otherwise pending annotation is invalid
+                    // and shape buttons remain disabled
+                    this.pendingAnnotationSource.clear();
```

Comment:
This doesn't work correctly, yet. If I create a multi-frame annotation and, for example, create two valid polygons first and then an invalid polygon, all three pending annotations are cleared where only the last one should disappear so I can try again.

See: [Screencast from 05.03.2024 09:18:13.webm](https://github.com/biigle/core/assets/2457311/496dd253-1ec9-4b48-aa5c-2359c03247dc)